src/postgres/event-handlers/CommentShared.ts

```diff
@@ -1,24 +1,27 @@
-import { CommentId } from '@subsocial/types/substrate/interfaces/subsocial';
+import { PostId, Post } from '@subsocial/types/substrate/interfaces/subsocial';
 import { substrate } from '../../substrate/subscribe';
 import { insertActivityForComment } from '../insert-activity';
-import { fillNotificationsWithAccountFollowers, fillNotificationsWithCommentFollowers } from '../fill-activity';
-import { SubstrateEvent, EventHandlerFn } from '../../substrate/types';
+import { fillNotificationsWithAccountFollowers, fillNotificationsWithCommentFollowers, fillNewsFeedWithBlogFollowers, fillNewsFeedWithAccountFollowers } from '../fill-activity';
+import { SubstrateEvent } from '../../substrate/types';
 
-export const onCommentShared: EventHandlerFn = async (eventAction: SubstrateEvent) => {
+export const onCommentShared = async (eventAction: SubstrateEvent, comment: Post) => {
   const { data } = eventAction;
```

Comment:
Fix this copypasta on all post events handlers:

```typescript
const { data } = eventAction;
const follower = data[0].toString();
const postId = data[1] as PostId;
```

it should be something like this everywhere:

```typescript
const { author, postId } = parsePostEvent(eventAction);
```